packages/main/src/components/media-chat/video/VideoRoom.jsx

```diff
@@ -0,0 +1,126 @@
+import { useState, useEffect } from "react";
+import AgoraUIKit from "agora-react-uikit";
+import { AGORA_APP_ID, AGORA_TOKEN } from "@zuri/utilities";
+import styles from "./styles/videoRoom.css";
+import axios from "axios";
+
+const VideoRoom = ({ workspaceId }) => {
+  const [videoCall, setVideoCall] = useState(false);
+  const [token, setToken] = useState(null);
+  const fetchToken = async () => {
+    const response = await fetch(
+      `https://staging.api.zuri.chat/rtc/${workspaceId}/publisher/userAccount`
```

Comment:
Import BASE_API_URL from Zuri utilities instead of hardcoding it like this. Because there's different base urls for the different development environments

Reply:
The endpoint is on staging, not on the main yet

Reply:
Is this the zc_core PR or what are you talking about?

Reply:
The api for fetching the token is currently on zc_core and currently the changes is hosted on staging.api.zuri.chat and not on the base url which is api.zuri.chat